netbox_netprod_importer/vendors/cisco/nxos.py

```diff
@@ -46,13 +47,11 @@ def _parse_type_from_transceiver(self, interface):
         raise TypeCouldNotBeParsedError()
 
     def _get_transceiver_by_if(self):
-        cmd = "show interface transceiver | json"
+        cmd = "show interface transceiver" + self._driver_end_selection()
 
         if not self.cache.get("transceivers"):
             transceiver_conf_dump = self.device.cli([cmd])[cmd]
-            transceivers = json.loads(
-                transceiver_conf_dump
-            )["TABLE_interface"]["ROW_interface"]
+            transceivers = self._correct_and_convert_to_dict(transceiver_conf_dump)["TABLE_interface"]["ROW_interface"]
```

Comment:
I set the limit of characters to 79char/line for this project.
Could you split it as before to respect this limitation please?

Reply:
Of course I can, I just use the PEP8 standards.
If for you this is fundamentally changed.

Reply:
Yes please ;)

Reply:
[Here is a commit](https://github.com/Anthony25/netbox-netprod-importer/pull/15/commits/8bf0fe6a63e6207bfd1b8d361198a4f47196d195)